evalml/pipelines/utils.py

```diff
@@ -385,6 +386,7 @@ def _make_pipeline_time_series(
            Valid options are "DatetimeFeaturizer", "EmailFeaturizer", "URLFeaturizer", "NaturalLanguageFeaturizer", "TimeSeriesFeaturizer"
         include_decomposer (bool): For time series regression problems, whether or not to include a decomposer in the generated pipeline.
             Defaults to True.
+        features (bool): Whether to add a DFSTransformer component to this pipeline.
```

Comment:
Just wondering if this param should be called something like `add_dfs_transformer` instead of `features` to make it a bit more clear? I guess this is consistent with what is used in `make_pipeline` though.